### ISSUE TITLE
Support ability to terminate previous invocation of command on new event

### ISSUE THREAD

**Issue by F21** [radovskyb/watcher#59]
It would be nice if there is an option to terminate the previous invocation of the command when the command is still executing and a file changes. This would be extremely useful when the command is a task that takes a while to complete and we do not wish to wait for it to finish and want to invoke the command immediate again.

**Comment by ysmood** [comment]
@F21 Maybe you should give this tool a try: https://github.com/ysmood/gokit

**Comment by ysmood** [comment]
@abdullah2993 at least you need  to modify your script like this:

```
kill -$(cat /.pid)
```

The minus before the dollar sign is the point.

**Comment by abdullah2993** [comment]
@ysmood not really...it will work just fine as it is, this will send a `SIGTERM` to the processes but if you want to send `SIGKILL` then you can modify it to `kill -9 $(cat /.pid)`

**Comment by abdullah2993** [comment]
@ysmood you are right.
